Test/Web/FunctionalTests/TestApps/WebAppFW45Sampled/WebAppFW45Sampled.csproj

```diff
@@ -120,6 +112,9 @@
     <Content Include="packages.config">
       <SubType>Designer</SubType>
     </Content>
+    <None Include="$(SolutionDir)\..\test.common.packages.config">
```

Comment:
does it actually work? So two packages config will be used by single project?

Reply:
everything compiles and all the tests pass. that was my metric for success :)

Reply:
I don't understand how it can be. It may be build ordering thing when you got lucky. Hopefully those projects will be re-written on new csproj format and this will not be an issue any longer.

I'm ok to merge if you need this change.